docs/development/runtime-compilation.md

```diff
@@ -0,0 +1,33 @@
+---
+description: Compile your runtime
+---
+
+# Runtime Compilation
```

Comment:
I think that this information could be placed in the `subsembly compile` section